base_maintenance/views/maintenance_team_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ForgeFlow S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0) -->
+<odoo>
+    <record id="maintenance_team_view_form" model="ir.ui.view">
+        <field name="name">maintenance.team.form</field>
+        <field name="model">maintenance.team</field>
+        <field name="inherit_id" ref="maintenance.maintenance_team_view_form" />
+        <field name="arch" type="xml">
+            <field name="member_ids" position="before">
+                <field name="user_id" />
+            </field>
+            <xpath expr="//group[1]" position="after">
+                <group>
+                    <field
+                        name="description"
+                        placeholder="Description of the team's function"
+                    />
+                </group>
+            </xpath>
+        </field>
+    </record>
+    <!-- There is no search view to inherit, so creating it-->
```

Comment:
This search view is already added in Odoo's core so its not needed here. I just realized it was like this also in v13 and v12, so maybe we should do a PR fixing it.

Reply:
Done!